README.md

```diff
@@ -77,3 +78,7 @@ print(unit_alias('kft.lbf'))
 
 If you have suggestions for improvement or you found bugs,
 please don't hesitate to put them in the issue list.
+
+
+## License
+[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2FSchlumberger%2FUOM.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2FSchlumberger%2FUOM?ref=badge_large)
```

Comment:
Codacy has a fix for the issue: [Files should end with a single newline character](https://app.codacy.com/gh/Schlumberger/UOM/pullRequest?prid=9389293)
```suggestion
[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2FSchlumberger%2FUOM.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2FSchlumberger%2FUOM?ref=badge_large)

```